Config.h

```diff
@@ -6,6 +6,72 @@
 
 constexpr auto KEYMOD_NONE = 0;
 
+constexpr bool DEFAULT_FULLSCREEN_MODE = false;
+constexpr int DEFAULT_RESOLUTION_WIDTH = 800;
+constexpr int DEFAULT_RESOLUTION_HEIGHT = 600;
+constexpr int DEFAULT_SIZE = /*20*/ 70;
+constexpr int DEFAULT_SIZE_Z = /*6*/ 4;
+
+class Config {
+    // constexpr auto GFXMODE = GFX_AUTODETECT_WINDOWED;
+public:
+    enum trackingmode : uint8_t {
+        TRACKING_NONE,
+        TRACKING_CENTER,
+        TRACKING_FOCUS,
+
+        TRACKING_INVALID
+    };
+
+    bool closeOnEsc = true;
+    int defaultScreenWidth{ DEFAULT_RESOLUTION_WIDTH };
+    int defaultScreenHeight{ DEFAULT_RESOLUTION_HEIGHT };
+    bool Fullscreen{ DEFAULT_FULLSCREEN_MODE };
+    Crd3D defaultSegmentSize{
+        .x = DEFAULT_SIZE,
+        .y = DEFAULT_SIZE,
+        .z = DEFAULT_SIZE_Z
+    };
+    bool show_all_creatures = false;
+    int automatic_reload_time = 50;
+    int automatic_reload_step = 50;
+    bool debug_mode = false;
+    bool transparentScreenshots = false;
+    int lift_segment = 0;
+    int animation_step = 196;
+    bool verbose_logging = false;
+    trackingmode track_mode = TRACKING_FOCUS;
+    bool invert_mouse_z = false;
+    bool follow_DFcursor = true;
+    bool show_creature_names = false;
+    bool show_creature_moods = false;
+    bool show_creature_jobs = false;
+    uint8_t show_creature_professions;
+    bool names_use_nick = true;
+    bool names_use_species = true;
+    bool show_osd = false;
+    bool cache_images = false;
```

Comment:
I know this matches the default config, but I wonder why this is off by default

Reply:
Because Stonesense was written in 2012 when the average graphics card had maybe 2 GB of VRAM and a lot of DF players were likely using GPUs with less than 1GB of VRAM, often much less, and using the image cache on such systems would make the baby GPU cry sadly for mercy.